### ISSUE TITLE
Reverse delegation

### ISSUE THREAD

**Issue by ghost** [poweradmin/poweradmin#85]
Support for adding reverse delegation for smaller subnets. For example 0/26.1.0.192.in-addr.arpa. The problem seems to be / in the name and also creates problems when editing existing reverse zones containing / as well as adding new ones.

**Comment by xrh0905** [comment]
Looking forward to this feature!
I'm recently trying to use PowerDNS with DN42, where I need to add a zone that generate the following error:

---

Given hostname has too many slashes.

---

It would be nice to have a working WebUI with PowerDNS.

**Comment by edmondas** [comment]
Validations for ARPA zones were updated and the fix was added to the master branch. The next release will come at the end of the year.

<img width="647" height="420" alt="Image" src="https://github.com/user-attachments/assets/8130ce6d-923d-4503-8c4b-b02cf85cd8e0" />

**Comment by Potat0000** [comment]
~~Hi! Just want to confirm. After 8928b82, can DNSSEC for RFC 2317 Classless rDNS domains function normally (e.g., display DS and DNSKEY records)? I just tested using [4.1.0 Pre-release](https://github.com/poweradmin/poweradmin/releases/tag/v4.1.0), and it appears DNSSEC is still not working properly.~~

Opened a new issue #994 to track this.

**Comment by edmondas** [comment]
@Potat0000 Thanks for testing! The v4.1.0 release had the zone creation fix but was missing URL encoding for the PowerDNS API calls - the `/` in RFC 2317 zone names was breaking the API endpoint paths for DNSSEC operations (cryptokeys, etc.).

This has been fixed in v4.1.1 (released Feb 15). Could you update and confirm DNSSEC works correctly for your classless rDNS zones?

**Comment by Potat0000** [comment]
> This has been fixed in v4.1.1 (released Feb 15). Could you update and confirm DNSSEC works correctly for your classless rDNS zones?

Yes, this has been fixed and now DNSSEC works correctly. Thank you for your work!
